src/featureParameters/P10_Application_Date.featureParameterDate

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FeatureParameterDate xmlns="http://soap.sforce.com/2006/04/metadata">
+    <dataflowDirection>LmoToSubscriber</dataflowDirection>
+    <masterLabel>Application to P10 program date retrieved from BizTech</masterLabel>
+    <value>2020-10-23</value>
```

Comment:
Why is this using a relatively arbitrary default value?

Reply:
@dcaballeroSf Was there a reason this is being populated with a default value? I would expect it to default to an empty/null value and then be populated through the LMO.